kubernetes/samples/scripts/create-weblogic-domain/model-in-image/README.md

```diff
@@ -430,7 +437,7 @@ To allow model-in-image to access the RCU database and OPSS wallet, it's necessa
 | `create_opss_key_secret.sh` | Defines secret `sample-domain1-opss-key-passphrase-secret` with `passphrase=welcome1` |
 | `create_rcu_access_secret.sh` | Defines secret `sample-domain1-rcu-access` with appropriate values for attributes `rcu_prefix`, `rcu_schema_password`, `rcu_admin_password`,  and `rcu_db_conn_string` |
 | `model1.yaml.jrf` | Populates the `domainInfo -> RCUDbInfo` stanza `rcu_prefix`, `rcu_schema_password`, `rcu_admin_password`,  and `rcu_db_conn_string` attributes by referencing their locations in the `sample-domain1-rcu-access` secret. |
-| `k8s-domain.yaml.template` | Ensures the domain mounts the OPSS key secret by setting the domain resource `opssWalletSecret` attribute to `sample-domain1-rcu-access`, and ensures the domain mounts the RCU access secret `sample-domain1-rcu-access` for reference by WDT model macros by setting the domain resource `configOverridesSecrets` attribute. |
+| `k8s-domain.yaml.template` | Ensures the domain mounts the OPSS key secret by setting the domain resource `configuration.opss.walletSecret` attribute to `sample-domain1-rcu-access`, and ensures the domain mounts the RCU access secret `sample-domain1-rcu-access` for reference by WDT model macros by setting the domain resource `configuration.secrets` attribute. |
 
```

Comment:
I think we agreed to change this to walletFileSecret and walletPasswordSecret per slack thread https://proddev-paas-fmw.slack.com/archives/CKFA73SNM/p1580504991015300